pom.xml

```diff
@@ -222,7 +222,7 @@
 			<dependency>
 				<groupId>mysql</groupId>
 				<artifactId>mysql-connector-java</artifactId>
-				<version>8.0.19</version>
+				<version>8.0.23</version>
```

Comment:
Let's don't upgrade the MySQL version. There is a breaking change in the MySQL JDBC driver which no longer works with Liquibase,

Reply:
@filiphr Good catch; I've reverted the change.